Jellyfin.Plugin.KodiSyncQueue/API/KodiSyncQueueController.cs

```diff
@@ -41,6 +41,7 @@ public KodiSyncQueueController(ILogger<KodiSyncQueueController> logger, IJsonSer
         /// <param name="filter">Comma separated list of Collection Types to filter (movies,tvshows,music,musicvideos,boxsets. The filter query must be lowercase in both the name and the items</param>
         /// <returns>The <see cref="SyncUpdateInfo"/>.</returns>
         [HttpGet("Jellyfin.Plugin.KodiSyncQueue/{userID}/GetItems")]
+        [HttpGet("/Jellyfin.Plugin.KodiSyncQueue/{userID}/GetItems")] // TODO: Fix the http mess in Kodi addon, then remove these again
```

Comment:
This probably won't work if there isn't a BaseUrl set

Reply:
```
[13:13:59] [ERR] [50] Jellyfin.Server.Middleware.ExceptionMiddleware: Error processing request. URL GET /api-docs/openapi.json.
Swashbuckle.AspNetCore.SwaggerGen.SwaggerGeneratorException: Conflicting method/path combination "GET Jellyfin.Plugin.KodiSyncQueue/{userID}/GetItems" for actions - Jellyfin.Plugin.KodiSyncQueue.API.KodiSyncQueueController.Ge
tLibraryItemsQuery (Jellyfin.Plugin.KodiSyncQueue),Jellyfin.Plugin.KodiSyncQueue.API.KodiSyncQueueController.GetLibraryItemsQuery (Jellyfin.Plugin.KodiSyncQueue). Actions require a unique method/path combination for Swagger/O
penAPI 3.0. Use ConflictingActionsResolver as a workaround
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.GenerateOperations(IEnumerable`1 apiDescriptions, SchemaRepository schemaRepository)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.GeneratePaths(IEnumerable`1 apiDescriptions, SchemaRepository schemaRepository)
   at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.GetSwagger(String documentName, String host, String basePath)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.ResponseCompression.ResponseCompressionMiddleware.Invoke(HttpContext context)
   at Jellyfin.Server.Middleware.ResponseTimeMiddleware.Invoke(HttpContext context) in C:\Users\Cody\Code\Jellyfin\jellyfin-master\Jellyfin.Server\Middleware\ResponseTimeMiddleware.cs:line 63
   at Jellyfin.Server.Middleware.ExceptionMiddleware.Invoke(HttpContext context) in C:\Users\Cody\Code\Jellyfin\jellyfin-master\Jellyfin.Server\Middleware\ExceptionMiddleware.cs:line 55
```

Reply:
Right, I'll just fix that client side then.